drivers/volume/volume.go

```diff
@@ -129,6 +129,10 @@ type Driver interface {
 	// GetReplicaSets returns the replica sets for a given volume
 	GetReplicaSets(vol *Volume) ([]*api.ReplicaSet, error)
 
+	//GetReplicatNodeSets returns the replica grouped per set for a given volume
```

Comment:
Delete these commented out lines.  If you need them they should be uncommented.

Reply:
Removed the commented lines